locustfile.py

```diff
@@ -0,0 +1,26 @@
+from locust import HttpUser, task, between
+
+
+class ChatUser(HttpUser):
+    wait_time = between(60,61)
+
+    def on_start(self):
+        self.client.get("/")
+
+    @task
+    def ask_question(self):
+        self.client.post("/chat", json={
+            "history": [
+                {
+                "user": "What is included in my Northwind Health Plus plan that is not in standard?"
+                }
+            ],
+            "approach": "rrr",
+            "overrides": {
+                "retrieval_mode": "hybrid",
+                "semantic_ranker": True,
+                "semantic_captions": False,
+                "top": 3,
+                "suggest_followup_questions": False
+            }
+            })
```

Comment:
Does the client make any requests after this?

Reply:
I've added a follow-up request with more history. I also moved everything into a single task. Seemed like better behavior modeling. In future, one could also have an /ask task, but just checking /chat reveals a lot of issues.